.travis.yml

```diff
@@ -17,3 +18,4 @@ script:
   - npm test
   - cd ..
   - bash ./deploy.sh
+  - curl -O https://sideshowbarker.net/nightlies/jar/vnu.jar && /usr/lib/jvm/java-8-oracle/jre/bin/java -jar vnu.jar --skip-non-html /home/travis/build/whatwg/streams/streams.spec.whatwg.org
```

Comment:
Does this mean that even with install we'd install it each time? Or this a more wasteful setup?